hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/client/HdfsClientConfigKeys.java

```diff
@@ -141,6 +141,8 @@
       "dfs.short.circuit.shared.memory.watcher.interrupt.check.ms";
   int     DFS_SHORT_CIRCUIT_SHARED_MEMORY_WATCHER_INTERRUPT_CHECK_MS_DEFAULT =
       60000;
+  String DFS_SHORT_CIRCUIT_NUM = "dfs.short.circuit.num";
```

Comment:
can i ask you to change this to dfs.client.short.circuit.num? all client configs starts with dfs.client

Reply:
if you change, please also update hdfs-default.xml

Reply:
@jojochuang thank you for interesting! Done)